model/jpa/src/main/resources/META-INF/jpa-changelog-2.3.0.xml

```diff
@@ -36,6 +36,16 @@
          <addColumn tableName="IDENTITY_PROVIDER">
              <column name="PROVIDER_DISPLAY_NAME" type="VARCHAR(255)"></column>
          </addColumn>
+
+         <addColumn tableName="REALM">
+             <column name="SSO_MAX_LIFESPAN_REMEMBER_ME" type="INT" defaultValueNumeric="36000"/>
```

Comment:
I don't like having a default for this. Rather values for remember me should be optional and it should use non-remember me values if not specified.